bentoml/_internal/runner/runner.py

```diff
@@ -208,6 +237,8 @@ class RunnerState(enum.IntEnum):
     INIT = 0
     SETTING = 1
     SET = 2
+    SHUTIING_DOWN = 3
+    SHUTDOWN = 4
```

Comment:
Not related in this PR, suggest renaming the two states above:
```
UNINITILIZED = 0
INITIALIZING = 1
READY = 2
```

And should it also have an `ERROR` state? in case `_setup` failed in runner_app?

Reply:
The names reflect our `setup` and `shutdown` API.

Reply:
But `READY` lgtm